networkx/generators/random_graphs.py

```diff
@@ -127,6 +134,9 @@ def gnp_random_graph(n, p, seed=None, directed=False):
         See :ref:`Randomness<randomness>`.
     directed : bool, optional (default=False)
         If True, this function returns a directed graph.
+    create_using : Graph, optional (default DiGraph() if directed else Graph())
+        If provided this graph is cleared of nodes and edges and filled
+        with the new graph. Usually used to set the type of the graph.
```

Comment:
The language in the doc_string should mimic that in other modules (where that makes sense).
That language looks like:
```
    create_using : NetworkX graph constructor, optional (default=nx.Graph)
       Graph type to create. If graph instance, then cleared before populated.
       Only undirected Graphs are supported.
```
but we should change it here I believe. I think the point of this change is that it doesn't need to be a NetworkX graph constructor, so I've removed "NetworkX". We need to describe both default values, so add that. And state that multigraph is not supported. I came up with the following:
```suggestion
    create_using : Graph constructor, optional (default=nx.Graph or nx.DiGraph)
        Graph type to create. If graph instance, then cleared before populated.
        Multigraph types are not supported and raise a ``ValueError``.
        By default NetworkX Graph or DiGraph are used depending on `directed`.
```
However we word it, this should be copied (as you already have) to all the other functions where `create_using` is added.